src/containers/IngredientBoard.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <main
+    class="flex h-full w-full flex-1 flex-col justify-start gap-y-5 px-5 pt-[34px]"
+  >
+    <div class="flex flex-wrap gap-x-2">
+      <button
+        :key="ingredient.id"
+        class="flex flex-col items-center justify-between gap-y-[10px]"
+        @click="
+          () => {
+            handleIngredientClick(ingredient.id);
+          }
+        "
+        v-for="ingredient in ingredients"
+      >
+        <div :class="getUserSelectStyle(ingredient)">
+          <img
+            :src="ingredient.imageUrl ?? PLACE_HOLD_IMAGE_URL"
+            :alt="ingredient.name"
+            width="56"
+            height="56"
+            class="rounded-full"
+          />
+        </div>
+        <p class="text-center text-small">{{ ingredient.name }}</p>
+      </button>
+    </div>
+  </main>
+</template>
+
+<script setup lang="ts">
+import { Ingredient } from '@src/interface/goods';
+import { useUserSelectStore } from '@src/store/storeUserSelect';
+import { PLACE_HOLD_IMAGE_URL } from '@src/constants';
```

Comment:
이렇게 사용하시면 됩니다. 이유는 tsconfig.json 파일을 참고해보세요!

```suggestion
import { Ingredient } from '@interface/goods';
import { useUserSelectStore } from '@store/storeUserSelect';
import { PLACE_HOLD_IMAGE_URL } from '@constants';
```